### DISCUSSION TITLE
Missing scrollbar component

### DISCUSSION THREAD

**Discussion by marektonik** [twilio-labs/paste#3393] [Help: Figma] [answered]
hey team, we can't find scrollbar component anywhere in the DS, could you guys give us some guidance how to indicate developers and users that some container is scrollable? I assume that most of the time users use native browser scrollbar but it's hidden quite often. In our case there is a modal with scrollable modal-body, we think it's better to have always-visible scrollbar, wdyt?

<img width="1875" alt="Evergreen - Unify - Profile settings - Edit - 11" src="https://github.com/twilio-labs/paste/assets/140410476/14ac46f9-090c-4284-a94d-c6ebbd216c3f">

**Comment by loreina** [comment, marked as answer]
Hi @marektonik,

We support native browser scrolling which can be different depending on the user's browser and OS. For example, you can see the scrollbar in [Modal with overflowing body content](https://paste-storybook.twilio.design/?path=/story/components-modal--overflowing-body-content) depending on your system's settings. We also don't have plans to support custom scrolling for the time being. Thanks!

**Reply by marektonik** [reply]
Understand, thanks @loreina
